src/coreclr/jit/inline.cpp

```diff
@@ -507,11 +543,22 @@ void InlineContext::DumpXml(FILE* file, unsigned indent)
         }
 
         fprintf(file, "%*s<%s>\n", indent, "", inlineType);
-        fprintf(file, "%*s<Token>%08x</Token>\n", indent + 2, "", calleeToken);
+        fprintf(file, "%*s<Name>%s</Name>\n", indent + 2, "", buf);
```

Comment:
The replay policy XML parsing is sensitive to the order of these entries.

Can you verify inline replay still works?